core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/TelemetryConverter.java

```diff
@@ -438,7 +438,9 @@ private void setClusterLogsCollection(TelemetryRequest request, Features feature
     }
 
     private void setMonitoring(TelemetryRequest request, Features features) {
-        if (monitoringEnabled) {
+        // second condition is a temporary workaround, later we should check an entitlement instead to simplify logic
```

Comment:
you will need to decide, do you want PaaS users to be able to define the behaviour or you want to rely on entitlements (so you need to create a ticket to enable for a customer) - note that, for other telemetry features like cloud storage based logging / cluster deployment logs / workload analytics -> that can be updated by the customer (but for example metering is a must), if user can define it, in that case i would keep this to not be defined by the entitlement / but by the request...but actually depends on how you want to maintain it